editor/editor_system.cpp

```diff
@@ -859,9 +861,12 @@ Variant EditorSystem::saveAll(const Event* event)
 
 Variant EditorSystem::autoSave(const Event* event)
 {
-	PRINT("Auto-saving current scene...");
-	saveAll(nullptr);
-	return true;
+	if (m_Autosave)
```

Comment:
What happens when the game is running in the editor and autosave is turned on? My guess is it would try to autosave in the middle of the game, which is not quite right. 

Likely autosave should be turned off when a game is running in the editor and it's state should be recovered when the game ends.